parser.go

```diff
@@ -95,3 +105,23 @@ func (ctx *ParserContext) parseBasicTags() {
 		}
 	}
 }
+
+func validateMetaTag(tag *MetaTag) error {
+	if tag.Name == "" && tag.Property == "" {
+		return errors.New("meta tag must have either a name or property attribute")
```

Comment:
It is recommended not to use dynamic error. You should place this error in top of the file like:

```go
var (
     errTagInvalid = errors.New("tag is invalid")
)

// In here
return fmt.Errorf("%w: meta tag must have either a name or property attribute", errTagInvalid)
```

Another option is to make this function just return `bool` as we only need to know whether we should keep the tag or not.